service-base/src/main/java/io/narayana/lra/LRAConstants.java

```diff
@@ -77,6 +77,11 @@ public final class LRAConstants {
      */
     public static final String ENLIST_PARTICIPANT_CLIENT_MAX_RETRY = "lra.participant.client.max.retry";
 
+    /**
+     * Custom base URI if the client needs to override default base URI info taken from the Jakarta REST filters.
+     */
+    public static final String NARAYANA_LRA_BASE_URI_PROPERTY_NAME = "narayana.lra.base-uri";
```

Comment:
I like the prefix of narayana.lra and won't block the review on this. I would mention it might not be consistent with other ones in here starting "lra." but I am not aware of it being defined as a requirement how the prefix is structured.